ppafm/data.py

```diff
@@ -93,7 +80,7 @@ def _extract_zip(archive_path, target_dir):
         _extract_members(ft, members, target_dir)
 
 
-def download_dataset(name: str, target_dir: PathLike):
+def download_dataset(name: str, target_dir: Union[Path, str]):
```

Comment:
Do not understand this one, is it just that you have found it on the way?

Reply:
Yes, this is just a random small fix, nothing to do with the logging.